app/views/editions/_actions.html.erb

```diff
@@ -5,37 +5,52 @@
       <% if edition.persisted? %>
         <%= link_to "Preview", preview_content_model_url(guide), class: 'btn btn-default add-right-margin' %>
       <% end %>
+
+      <% if !publish_controls_only %>
+        <%= form.submit "Save", class: 'btn btn-default add-right-margin js-ok-to-navigate-away', name: :save, disabled: edition.unpublished?, data: disable_if_new_record(guide) %>
```

Comment:
Could the disabled and data-disabled be combined?

Actually - I don't think the data-disable is doing anything. On integration the button seems to be available when the record is new.